ci/gpu/build.sh

```diff
@@ -152,6 +152,13 @@ if [[ -z "$PROJECT_FLASH" || "$PROJECT_FLASH" == "0" ]]; then
             echo "Running GoogleTest $test_name"
             ${gt} --gtest_output=xml:"$WORKSPACE/test-results/"
         done
+
+        # Test libcudf (csv, orc, and parquet) with `LIBCUDF_CUFILE_POLICY=KVIKIO`
```

Comment:
To clarify my understanding of this change -- would running this test have caught the problem?

Reply:
Exactly, I had the CI run (and fail) before pushing the fix